mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -196,7 +199,7 @@ class SetpointRawPlugin : public MavRosPlugin,
 				position.x(), position.y(), position.z(),
 				velocity.x(), velocity.y(), velocity.z(),
 				af.x(), af.y(), af.z(),
-				req->yaw, req->yaw_rate);

```

Comment:
I just wanted to make the transformation really obvious, like the others. A library function would be good I think. Will add one and see how it looks, and also update the rest

> On 15.11.2015, at 19:16, Vladimir Ermakov <notifications@github.com> wrote:
> 
> In mavros/src/plugins/setpoint_raw.cpp:
> 
> > @@ -196,7 +199,7 @@ class SetpointRawPlugin : public MavRosPlugin,
> >  				position.x(), position.y(), position.z(),
> >  				velocity.x(), velocity.y(), velocity.z(),
> >  				af.x(), af.y(), af.z(),
> > -				req->yaw, req->yaw_rate);
> Why not:
> 
> -(req->yaw), -(req->yaw_rate)); // enu-ned transformation
> —
> Reply to this email directly or view it on GitHub.
>